aten/src/ATen/native/miopen/Conv_miopen.cpp

```diff
@@ -524,7 +524,20 @@ void miopen_convolution_add_bias_(CheckedFrom c, const TensorArg& output, const
   checkSize(c, bias, { output->size(output_channels_dim) });
 
   TensorDescriptor bdesc, odesc;
-  bdesc.set(bias->expand({1, bias->size(0)}), output->dim());
+
+  auto memory_format = output->suggest_memory_format();
+
+  std::vector<int64_t> shape( output->dim(), 1);
+  shape[output_channels_dim] = -1;
+  at::Tensor bias_contig =  bias->reshape(shape).contiguous(memory_format);
+  // Make sure that NC11 strides follow formula
+  bias_contig.resize_(bias_contig.sizes(), memory_format );
+
+  // TODO: Workaround since MIOpen does not support NHWC bias
```

Comment:
Note: Internal tickets opened against frameworks pytorch to fix this later: 
https://ontrack-internal.amd.com/browse/SWDEV-301466
See ticket for linked MIOpen ticket for NHWC bias support.

Reply:
link the ticket in the code